src/main/resources/templates/Index.html

```diff
@@ -1,85 +1,87 @@
 <!DOCTYPE html>
-<html xml:th="http://www.thymeleaf.org">
+<html xmlns:th="http://www.thymeleaf.org" lang="en-US">
 <head>
-  <link rel="stylesheet" href="index.css">
+  <title>Voluntier</title>
+  <link rel="stylesheet" th:href="@{css/Index.css}">
+
   <script>
     function openForm() {
       document.getElementById("myForm1").style.display = "block";
     }
 
-    function closeForm() {
-      document.getElementById("myForm").style.display = "none";
+    function closeForm1() {
+      document.getElementById("myForm1").style.display = "none";
+    }
+
+    function closeForm2() {
+        document.getElementById("myForm2").style.display = "none";
     }
   </script>
 </head>
 <body>
 <div class="content">
   <header>
-    <img src="images.png" alt="Logo">
-
+    <img th:src="@{/images/images.png}" alt="Logo">
   </header>
+
   <div class="tab">
-    <button><a href="index.html">About this App</a></button>
+    <button><a href="/">About this App</a></button>
```

Comment:
Fixed href attributes by referencing endpoints defined in Controller as applicable.